stackstorm-compose/docker-compose.yml

```diff
@@ -0,0 +1,202 @@
+    networks:
+      - private
+      - public
+    dns_search: .
+  st2api:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2api:${ST2_VERSION:-3.3dev}
+    depends_on: ["mongo", "rabbitmq"]
+    networks:
+      - private
+    environment:
+      ST2_AUTH_URL: ${ST2_AUTH_URL:-http://st2auth:9100/}
+      ST2_API_URL: ${ST2_API_URL:-http://st2api:9101/}
+      ST2_STREAM_URL: ${ST2_STREAM_URL:-http://st2stream:9102/}
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+      - stackstorm-packs-configs:/opt/stackstorm/configs:rw
+      - stackstorm-packs:/opt/stackstorm/packs:rw
+      - ${ST2_PACKS_DEV:-./packs.dev}:/opt/stackstorm/packs.dev:rw
+    dns_search: .
+  st2stream:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2stream:${ST2_VERSION:-3.3dev}
+    depends_on: ["st2api"]
+    networks:
+      - private
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+    dns_search: .
+  st2scheduler:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2scheduler:${ST2_VERSION:-3.3dev}
+    depends_on: ["st2api"]
+    networks:
+      - private
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+    dns_search: .
+  st2workflowengine:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2workflowengine:${ST2_VERSION:-3.3dev}
+    depends_on: ["st2api"]
+    networks:
+      - private
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+    dns_search: .
+  st2auth:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2auth:${ST2_VERSION:-3.3dev}
+    depends_on: ["st2api"]
+    networks:
+      - private
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+      - ./htpasswd:/etc/st2/htpasswd:ro
+    dns_search: .
+  st2actionrunner:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2actionrunner:${ST2_VERSION:-3.3dev}
+    depends_on: ["st2api"]
+    networks:
+      - private
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+      - stackstorm-packs-configs:/opt/stackstorm/configs:rw
+      - stackstorm-packs:/opt/stackstorm/packs:rw
+      - ${ST2_PACKS_DEV:-./packs.dev}:/opt/stackstorm/packs.dev:rw
+      - stackstorm-virtualenvs:/opt/stackstorm/virtualenvs:rw
+      - stackstorm-ssh:/home/stanley.ssh
+    dns_search: .
+  st2garbagecollector:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2garbagecollector:${ST2_VERSION:-3.3dev}
+    depends_on: ["st2api"]
+    networks:
+      - private
+    volumes:
+      - ./st2.docker.conf:/etc/st2/st2.docker.conf:ro
+      - ./st2.user.conf:/etc/st2/st2.user.conf:ro
+    dns_search: .
+  st2notifier:
+    image: ${ST2_IMAGE_REPO:-stackstorm/}st2garbagecollector:${ST2_VERSION:-3.3dev}
```

Comment:
Will `st2notifier` be added later?

Reply:
Fixed the image name for st2notifier.  Thank you!